src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
+	register "sata_ports_enable[1]" = "1" # M.2 2280
+	register "sata_ports_dev_slp[0]" = "1"
+	register "sata_ports_dev_slp[1]" = "1"
+
+	device domain 0 on
+		device ref igpu on
+			subsystemid 0x8086 0x2212
+			register "ddi_portA_config" = "1" # HDMI on port A
+			register "ddi_portB_config" = "1" # DP on port B
+			register "ddi_ports_config" = "{
+				[DDI_PORT_A] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_B] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_1] = DDI_ENABLE_HPD,
+				[DDI_PORT_2] = DDI_ENABLE_HPD,
+			}"
+		end
+
+		# SATA
+		device ref pcie_rp11 off end
+		device ref pcie_rp12 off end
+		# LAN1
+		device ref pcie_rp2 on
+			register "pch_pcie_rp[PCH_RP(2)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
```

Comment:
Don't disable ASPM, let coreboot decide whether to enable or disable ASPM.

Reply:
`.PcieRpL1Substates = L1_SS_DISABLED,` is also an ASPM substate.

You mean Linux? Or both WIndows and Linux? I think Windows will have problems regardless of ASPM state.

Reply:
@miczyg1 Linux; i get 

```
[  609.931277] igc 0000:01:00.0 enp1s0: PCIe link lost, device now detached
[  625.931534] igc 0000:04:00.0 enp4s0: PCIe link lost, device now detached
[  673.933508] igc 0000:02:00.0 enp2s0: PCIe link lost, device now detached
```
when trying to plug in an ethernet cable into either of the ports, whenever ASPM is not explicitly disabled. Then, I tried to provide fake clkreqs as you did for [VP66XX](https://github.com/Dasharo/coreboot/pull/574/commits/761db8e8a040ea08d2e8d7e91bf0f4a705764a0e), but it didn't help

and yes, both L1 and ASPM are enabled now in the [overridetree](https://github.com/Dasharo/coreboot/pull/576/files#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2R27)

Reply:
Try with ASPM disabled and see if Windows BSODs when you try to install drivers for the Ethernet ports. If it doesn't;t BSOD the new can keep ASPM disabled. Otherwise we will need a fix in FSP like I mentioned elsewhere.

Reply:
Yeah it BSODs on startup with `CLOCK_WATCHDOG_TIMEOUT`, while Linux works seamlessly after disabling ASPM. I would really appreciate some info on how to apply the fix in FSP

Reply:
I would like to know that too. I haven't analyzed it yet how we can fix this. There is also licensing aspect.

Reply:
https://github.com/Dasharo/coreboot/pull/576/commits/d25d148fd1f1de1129b31005f821f5b94ca00354

Added custom FSP with the missing parameters.